### ISSUE TITLE
'wsse:FailedAuthentication - Profile accrual is required

### ISSUE THREAD

**Issue by dukv2001** [Terrance/SkPy#141]
'wsse:FailedAuthentication - Profile accrual is required

**Comment by Terrance** [comment]
If this is an an ex-Skype account with an old username, have you tried using both the Skype username and the associated Microsoft account email address / phone number as the login username?

**Comment by Terrance** [comment]
There are similar account issues being reported near the end of EionRobb/skype4pidgin#656, with a few potential solutions.

**Comment by suurjaak** [comment]
> If this is an an ex-Skype account with an old username, have you tried using both the Skype username and the associated Microsoft account email address / phone number as the login username?

Yes it's an original Skype account. Trying with the associated e-mail address made no difference, still "Profile accrual is required". 

> There are similar account issues being reported near the end of EionRobb/skype4pidgin#656, with a few potential solutions.

Well! [This suggestion](https://github.com/EionRobb/skype4pidgin/issues/656#issuecomment-677788102) worked for me - enabling Outlook for my account. That resulted in the e-mail address becoming the primary alias for the account, and skpy login now works, both with skypename and e-mail. 

So apparently the new requirement is that the primary alias needs to be an e-mail, with skypename becoming deprecated.

**Comment by annalitvin** [comment]
@Terrance 
I get an error message:
> skpy.core.SkypeAuthException: ('wsse:FailedAuthentication - Authentication Failure', <Response [200]>)

But the message is being sent to the chat.
Code:
```
sk = Skype('email', 'password') # connect to Skype
ch = sk.chats.chat('chat_id")
ch.sendMsg("Some message")
```
Using online version of Skype. At the same time, the desktop version is installed on ubuntu 19.10.

**Comment by Terrance** [comment]
Have you tried any of the solutions on the issue linked above?

> But the message is being sent to the chat.

You can't do anything until you're authenticated with a registration token, so I don't know what that means.

**Comment by annalitvin** [comment]
> Have you tried any of the solutions on the issue linked above?

Set email as the main login alias.
> You can't do anything until you're authenticated with a registration token, so I don't know what that means.

Sorry, there was an error in the code - an attempt was made to log in twice with different passwords.
